source/slang/slang-emit-c-like.cpp

```diff
@@ -2620,6 +2629,51 @@ void CLikeSourceEmitter::_emitInst(IRInst* inst)
     }
 }
 
+void CLikeSourceEmitter::emitStore(IRStore* store)
+{
```

Comment:
Seems a bit fragile. Code where the var isn't the immediately before the store seems possible.

Reply:
It *is* a bit fragile, but it is also the kind of logic that was already in place.